tests/reftests/admin.test

```diff
@@ -654,9 +654,83 @@ opam-version: "2.0"
 ### ::::::::::::::::::::::::::::::::::::::::::::::::::::::
 ### :: Cache is tested in admin-cache.test ::
 ### ::::::::::::::::::::::::::::::::::::::::::::::::::::::
-### :
+### ::::::::::::::
+### :VII: Filter :
```

Comment:
i'd prefer if the new sections added were at the end for the same reason discussed in https://github.com/ocaml/opam/pull/6135#pullrequestreview-2349433012